packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/saml2/appidpinokta.md

```diff
@@ -18,8 +18,8 @@ We recommend that if the external IdP requires information from Okta for setup b
 
 * **IdP Issuer URI** &mdash; the issuer. The Identity Provider provides this value.
 
-* **IdP Single Sign-On URL** &mdash; the sign-on URL from the Identity Provider. If you sign the authN request by selecting the **Request Signature** option, but don't specify a destination in the **Destination** field (see [Advanced Settings](/docs/reference/social-settings/)), Okta automatically sends the authN request to the Identity Provider Single Sign-On URL.
+* **IdP Single Sign-On URL** &mdash; the sign-on URL from the Identity Provider. If you sign the authN request by selecting the **Request Signature** option, but don't specify a destination in the **Destination** field (see [Advanced Settings](#social-identity-provider-settings)), Okta automatically sends the authN request to the Identity Provider Single Sign-On URL.
```

Comment:
```suggestion
* **IdP Single Sign-On URL** &mdash; The sign-on URL from the Identity Provider. If you sign the authN request by selecting the **Request Signature** option but don't specify a destination in the **Destination** field (see [Advanced Settings](#social-identity-provider-settings)), Okta automatically sends the authN request to the Identity Provider Single Sign-On URL.
```

Reply:
I'm going to leave the lower case to be consistent with the rest of the bullet descriptions.